deploy/operator/base/rbac.yaml

```diff
@@ -1,130 +1,6 @@
-apiVersion: rbac.authorization.k8s.io/v1
```

Comment:
Just something to be aware of is that this change will mean we need to also do a corresponding update in `pre-kcp` infra-deployments to make sure the operator still has permission.

Depending on when this goes in this may or may not be an issue.

Reply:
yeah generally speaking (and by that I mean the best practice we adopted in openshift) it is better to have more files with single instances of api obj instances vs. one of these yaml files with a bunch of objects, and so I was trying to handle that here; however, also at the time (about 2 weeks ago now) I first did this break up, I think I was getting weird errors trying to execute the new kcp kustomize paths when access this global file (I don't have those errors now however 2 weeks later) - complaints about creating a file twice or something, don't remember for sure now.

however, the amount of rbac changes appears active (I see you have more changes in your sidecar / cache refactor).  It may make sense to break up the rbac.yaml file in a separate PR when things have settled down (assuming I do not hit that same error I saw 2 weeks ago with the kcp kustomzie apply).

I hope to circle back to this PR today, and we'll try to go back to just an rbac.yaml file, and see what issues I do or do not hit.  If I don't hit any issues, then I'll stay with rbac.yaml and we can consider breaking it up to "adhere to best practices" in a separate PR.

Reply:
ok I was able to sort this out / get `rbac.yaml` working

will be pushing shortly